addons/point_of_sale/static/src/app/store/pos_store.js

```diff
@@ -196,6 +335,42 @@ export class PosStore extends Reactive {
             return "flex-row-reverse justify-content-between m-1";
         }
     }
+    getProductPriceFormatted(product) {
+        const currency = this.config.currency_id;
+        const formattedUnitPrice = formatCurrency(this.getProductPrice(product), currency);
```

Comment:
```suggestion
        const formattedUnitPrice = this.env.utils.formatCurrency(this.getProductPrice(product));
```

If this recommendation doesn't work, may I know why?